src/style.css

```diff
@@ -1,3 +1,74 @@
+@font-face {
+  font-family: Roboto;
+  src: url(fonts/Roboto/Roboto-Medium.ttf);
+}
+
 body {
   margin: 0;
 }
+
+ul {
+  list-style-type: none;
+  margin: 0;
+  padding: 0;
+}
+
+li {
+  display: block;
+  float: right;
+}
+
+li a {
+  text-align: center;
+}
+
+.header {
+  width: 100%;
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
+}
+
+.nav {
+  flex-direction: row-reverse;
+}
+
+.link-8,
+.link-5 {
+  color: #00acdc;
+}
+
+.link {
+  text-decoration: none;
+  color: black;
+  font-family: Roboto, sans-serif;
+  height: 60px;
+  display: flex;
+  font-size: 12px;
+  text-align: center;
+  align-items: center;
+  justify-content: center;
+  padding: 0 20px;
```

Comment:
first and last link shouldn't have padding on nav edge 
![image](https://github.com/user-attachments/assets/f17e7dfc-dab9-4c48-a933-b0f16d05c273)
![image](https://github.com/user-attachments/assets/5866d80e-bdd5-448a-8a2a-eab35020fc6f)